sharktank/README.md

```diff
@@ -10,6 +10,10 @@ This sub-project is a work in progress. It is intended to be a repository of
 layers, model recipes, and conversion tools from popular LLM quantization
 tooling.
 
+## Project Status
+
+[![CI - Perplexity](https://github.com/nod-ai/SHARK-Platform/actions/workflows/ci_eval.yaml/badge.svg?branch=main&event=schedule)](https://github.com/nod-ai/SHARK-Platform/actions/workflows/ci_eval.yaml)
```

Comment:
We can also discuss putting this in the repository README (https://github.com/nod-ai/SHARK-Platform?tab=readme-ov-file#sharktank), but I want to limit to only a few badges (and workflows). Seems fine to start with this in `/sharktank/README.md` instead of `/README.md`

Reply:
Yes, SHARK-Platform README was my initial choice, but with the recent badges there, didn't want to clutter it too much. 
Once our CIs and codebase matures, we would want e2e tests that cover export, compile, evaluate (perplexity) & maybe benchmark, to run as a single ci-sharktank.yml nightly and report on the repo's README as a single badge.